src/css/variables.scss

```diff
@@ -1,14 +1,15 @@
 $colorPrimary: #4d4e4f;
 $colorHeading: #ffffff;
 $colorBorder: black;
-$colorHydration: #5BC8AC;
-$colorSleep: #EC96A4;
-$colorActivity: #DFE166;
+$colorHydration: mix(white, #00AA00, 40%);
```

Comment:
Color functions are interesting. I like the effect they've already had on our site.